playbooks/roles/hypershift-agent/tasks/install.yml

```diff
@@ -0,0 +1,374 @@
+---
+
+- name: Install ACM/MCE operator
+  block:
+    - name: Install the CRD
+      shell: |
+        oc apply -f https://raw.githubusercontent.com/openshift/assisted-service/master/hack/crds/metal3.io_baremetalhosts.yaml
+        oc apply -f https://raw.githubusercontent.com/openshift/assisted-service/refs/heads/master/hack/crds/metal3.io_preprovisioningimages.yaml
+      ignore_errors: true
+
+    - name: Deploy ACM/MCE Operator
+      block:
+        - name: Setup custom catalog source for ACM
+          block:
```

Comment:
Can we use separate block for ACM and MCE installation ? or can we import directly instead of using this block because this block only imports one role.